src/DNA/native/src/JIT.c

```diff
@@ -132,8 +133,19 @@ static void PushU32_(tOps *pOps, U32 v, I32 opSequencePoint) {
 	if (pOps->ofs >= pOps->capacity) {
 		pOps->capacity <<= 1;
 		//dprintfn("a.pOps->p = 0x%08x size=%d", pOps->p, pOps->capacity * sizeof(U32));
-		pOps->p = realloc(pOps->p, pOps->capacity * sizeof(U32));
-		pOps->pSequencePoints = realloc(pOps->pSequencePoints, pOps->capacity * sizeof(U32));
+		pOps->p = realloc(pOps->p, pOps->capacity * sizeof(size_t));
```

Comment:
Shouldn't this be U32 here instead of size_t, since you introduced PushSizeT_ below?

Reply:
Nevermind, I see you changed the storage.